frontend/src/components/releaseEditor/ContributorEditForm.vue

```diff
@@ -111,7 +111,7 @@
     </form>
   </div>
   <div class="modal-footer border-0">
-    <button type="button" class="btn btn-outline-gray" @click="resetContributor">Reset</button>
+    <button v-if="!disableEditForm" type="button" class="btn btn-outline-gray" @click="resetContributor">Reset</button>
```

Comment:
IIRC this unfortunately makes the reset button disappear once you search for and select a user in the "Create New Contributor" pathway which the only use case you'd want for to begin with